WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -337,34 +337,61 @@ internal HdPubKey GenerateNewKey(LabelsArray labels, KeyState keyState, bool isI
 		}
 	}
 
-	public HdPubKey GetNextReceiveKey(LabelsArray labels)
+	public HdPubKey GetNextReceiveKey(LabelsArray labels, ScriptPubKeyType scriptPubKeyType = ScriptPubKeyType.Segwit)
```

Comment:
We don't call this method with `ScriptPubKeyType.TaprootBIP86` in the client, only in the test.

Reply:
I don't know how the UI will look like, most wallets allow you to chose the type of address you want to generate (no random, ofc not).